source/postprocess/dynamic_topography.cc

```diff
@@ -41,7 +41,8 @@ namespace aspect
                                quadrature_formula,
                                update_values |
                                update_gradients |
-                               update_q_points);
+                               update_q_points |
+                               update_JxW_values);
```

Comment:
This took me some digging through deal.II, but I think I found the reason. The finite element we use (`FE_Q`) requires, that when `update_gradients` is set, it also needs `update_covariant_transformation` from the mapping. `MappingQGeneric` the base class of our used `MappingQ1` then decides that it needs `update_JxW_values` when `update_covariant_transformation` is set, so it automatically adds it to the update flags. `MappingCartesian` on the other hand does not require `update_JxW_values` for `update_covariant_transformation`. I am not sure if the behavior of `MappingQGeneric` is correct, but I can see that MappingCartesian only needs to scale the gradient from the reference cell to the real cell, and so might need less additional information than `MappingQGeneric`